File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        String total1 ="";
        String total2 ="";
        int answer = 0;
        for(int i = 0; i< num_list.length; i++){
            if(num_list[i] % 2 == 0){
                total1 += String.valueOf(num_list[i]);
            }
        }
          for(int i = 0; i< num_list.length; i++){
             if(num_list[i] % 2 != 0){
                total2 += String.valueOf(num_list[i]);
            }
        }
        int total3 = Integer.parseInt(total1);
        int total4 = Integer.parseInt(total2);
        answer = total3 + total4;
        
        return answer;
    }
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    # 변수 선언
    answer = 0 
    odd = ""
    even = ""
    # 반복문을 돌려 홀수 짝수를 구분 후 각 변수에 문자열로 더해준다.
    for i in num_list:
        if i % 2 != 0 :
            odd += str(i)
        else :
            even += str(i)
    # 다시 정수로 변환해 더해준다
    answer = int(odd) + int(even)
    return answer